sky/setup_files/MANIFEST.in

```diff
@@ -1,13 +1,11 @@
 include sky/backends/monkey_patches/*.py
 exclude sky/clouds/service_catalog/data_fetchers/analyze.py
 include sky/provision/kubernetes/manifests/*
+include sky/provision/azure/*
```

Comment:
Why do we need this for Azure only, while other clouds using new provisioner does not need this?

Reply:
I see. Got it!